### DISCUSSION TITLE
Problem eslint babel in new project quasar 2

### DISCUSSION THREAD

**Discussion by roylans** [quasarframework/quasar#10638] [CLI - PWA mode] [answered]
Hi.

I have created a new project with quasar 2 but I have this error in all templates

editor VSCode
![image](https://user-images.githubusercontent.com/16232825/132881378-b3339777-3a6b-44b9-937e-73e52c03fd92.png)

any suggestion
Gracias

**Comment by ontwikkelfabriek** [comment, marked as answer]
@roylans 

I would start here:
https://quasar.dev/start/vs-code-configuration
